src/Appwrite/Specification/Format/Swagger2.php

```diff
@@ -413,7 +413,11 @@ public function parse(): array
                         /** @var \Utopia\Validator\WhiteList $validator */
                         $node['type'] = $validator->getType();
                         $node['x-example'] = $validator->getList()[0];
-
+                        if (!($route->getLabel('sdk.namespace', '') == 'users' && $route->getLabel('sdk.method', '') == 'getUsage' && $name == 'provider')) {
```

Comment:
I don't think we should explicitly exclude the usage provider

Reply:
This was done because of [this](https://github.com/2002Bishwajeet/sdk-for-web/commit/9492f2f0914cc5e2600d448dcbefd87876d2a220#diff-a9c15c236009c0168cd4548b068c3a0994d8f3eb30f10995373cff94d6cf3d47)

So the files were getting overwritten and we didn't need the providers prefixed with Oauth-provider , thus excluded

Reply:
Let's try create a blacklist instead of specifically excluding a single route. That way it will be extensible if we need to exclude any others in future

Reply:
Done